packages/main/src/modules/cli.ts

```diff
@@ -27,10 +28,11 @@ async function getEnv(path: string) {
 }
 
 export async function init(path: string, repo?: string): Promise<void> {
-  const initCommand = run('@dcl/sdk-commands', 'sdk-commands', {
+  const initCommand = await run('@dcl/sdk-commands', 'sdk-commands', {
     args: ['init', '--yes', '--skip-install', ...(repo ? ['--github-repo', repo] : [])],
     cwd: path,
     env: await getEnv(path),
+    workspace: APP_UNPACKED_PATH + '/internal',
```

Comment:
how this will work for dev process like `npm start` ? maybe the other approach is better but there we need to install an extra dependency like npm

Reply:
if we go for this approach, maybe create an internal folder that holds the package.json and we need to copy that inside the asar package ? could that be possible ? 
just thinking loud and taking notes for tuesday

Reply:
but if we go this way, both production & dev packages would have the `/internal/node_modules` package so, in theory, will work